### PR TITLE
import-db: support importing a table

### PR DESCRIPTION
**Pull request by skshetry**
I have also made `import-db` command visible in the help message and hidden the dbt-specific options.

```console
$ dvc import-db --table customer --conn pgsql
```

(Note that I haven't added support for downloading a table with dbt's connection profile).

### CODE REVIEW

**Comment by dberenbaum** [comment]
Could be a separate PR, but should we drop the dbt part at this point?

File: dvc/commands/imp_db.py [hunk 1]
```diff
@@ -65,61 +72,87 @@ def run(self):
 
 class CmdImportDb(CmdBase):
     def run(self):
-        if not (self.args.sql or self.args.model):
-            raise argparse.ArgumentTypeError("Either of --sql or --model is required.")
+        from dvc.exceptions import InvalidArgumentError
+
+        if self.args.table or self.args.sql:
```
Comment (dberenbaum) [inline dvc/commands/imp_db.py:77]: Are one of these required?

Comment (skshetry) [reply dvc/commands/imp_db.py:77]: Yes they are required, and are mutually exclusive.

**Review by dberenbaum — APPROVED** [review]
LGTM but what do you think about dropping the dbt stuff at this point? Seems like it would be useful to document how you could connect your dbt etl to your dvc pipeline, but I don't see much benefit in keeping the dbt-based implementation here.

**Comment by skshetry** [comment]
> LGTM but what do you think about dropping the dbt stuff at this point? Seems like it would be useful to document how you could connect your dbt etl to your dvc pipeline, but I don't see much benefit in keeping the dbt-based implementation here.

I was thinking of dropping in the next release, but we could do it now. Will create a separate PR.

**Pull request merged**